<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultDialog</class>
 <widget class="QDialog" name="ResultDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>结果</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>851</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>插入</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
